sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage/src/Microsoft.Azure.WebJobs.Extensions.Storage.csproj

```diff
@@ -15,6 +15,9 @@
   </PropertyGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\Microsoft.Azure.WebJobs.Extensions.Storage.Blobs\src\Microsoft.Azure.WebJobs.Extensions.Storage.Blobs.csproj" />
-    <ProjectReference Include="..\..\Microsoft.Azure.WebJobs.Extensions.Storage.Queues\src\Microsoft.Azure.WebJobs.Extensions.Storage.Queues.csproj" />
+		<!-- Remove pointing to the released package of Microsoft.Azure.WebJobs.Extensions.Storage.Queues and reenable pointing to the unreleased version after
+		release the common package. https://github.com/Azure/azure-sdk-for-net/issues/38145 -->
```

Comment:
This will be removed after the release.
https://github.com/Azure/azure-sdk-for-net/issues/38145